spec/hardware/cpu.fmf

```diff
@@ -70,7 +70,7 @@ description: |
     are also useful resources.
 
     .. versionchanged:: 1.38
-       ``beaker`` plugins supports ``stepping``
+       ``beaker`` plugins supports ``frequency`` and ``stepping``
```

Comment:
Both release note and this `versionchanged` need an update, it's no longer 1.38 but 1.39.

Reply:
addressed